src/Widgets/AppContextMenu.vala

```diff
@@ -20,6 +20,7 @@ public class Slingshot.AppContextMenu : Gtk.Menu {
 
     public string desktop_id { get; construct; }
     public string desktop_path { get; construct; }
+    public DesktopAppInfo app_info { get; construct; }
```

Comment:
I don't think this needs to be public and it looks like this is assigned in the construct block, not as a constructor property.
```suggestion
    private DesktopAppInfo app_info;
```